### DISCUSSION TITLE
Trigger Multiple Events on a single Button

### DISCUSSION THREAD

**Discussion by jplewis01** [nguyenquyhy/Flight-Tracker-StreamDeck#307] [Ideas]
Is there a way using either a standard button OR Feedback value for a gauge  to trigger multiple events? 
One use case is the TBM Throttle, each setting for LOW IDLE, HIGH IDLE, and FLIGHT IDLE that requires specific Throttle axis and Propeller Axis values to be set, Technically it needs an additional local variable to guard against reverse thrust jumping the throttle over to the Feather side on the quadrant, tho there are other ways to mitigate that).

Possible Solution: Allow the user to enter multiple Event items separated by semi-colons or by Pipe symbols so you could do the following:  `AXIS_PROP_SET==2458 | AXIS_THROTTLE_SET==16383 | ENG_ANTI_ICE__1==1`

Phil

**Comment by DonQuilmi** [comment]
Late answer : use the multi-action command from the standard Stream Deck functions. It creates a folder where you can drag several toggle buttons.
